deployment/cpu-dev-ds.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apps/v1
 kind: DaemonSet
```

Comment:
cpu device daemon set also requires some proper role to be able to access Kubernetes "node" resource. Can this kind of change included here as well.
That change could fix this issue: https://github.com/nokia/CPU-Pooler/issues/19

Reply:
I'd rather keep changes to different issues in separate pull requests but to make the example deployment working, I'm adding the rbac configuration here.